cmd/ipfs/init.go

```diff
@@ -102,31 +109,41 @@ environment variable:
 			}
 		}
 
-		profile, _ := req.Options[profileOptionName].(string)
+		cfgLocation, _ := req.Options[configOptionName].(string)
+		profiles, _ := req.Options[profileOptionName].(string)
 
-		var profiles []string
-		if profile != "" {
-			profiles = strings.Split(profile, ",")
+		if cfgLocation != "" {
```

Comment:
Hm, it looks like we _currently_ accept one argument for the "initial config". We should:

1. Make sure we aren't passed both.
2. Consider deprecating that?

Or maybe we don't need this flag on init (we could _only_ support passing the config as an argument on init?). @hugomrdias, what's the state of javascript here?